### DISCUSSION TITLE
Recent systemd user services crashes

### DISCUSSION THREAD

**Discussion by rieje** [netblue30/firejail#5295] [Q&A]
I have some applications autostarted via systemd user services and recently (maybe a month or so?) these applications that firejail run crash seemingly randomly. I'm not too familiar with configuring firejail or reviewing systemd logs but `journalctl --user` shows among other things:

     firejail[2463]: Access error: uid 1000, last mount name:/ dir:/run/user/1000/gvfs type:fuse.gvfsd-fuse - invalid read-only mount

Restarting the service is no problem, I just have to do it every time it crashes (a few times a day). It happens most with signal-desktop but happened to deluge less frequently.

Any ideas?

EDIT: On startup at least almost always fails. At least for today:

    signal-desktop[5026]: [15:0805/090744.152931:ERROR:bus.cc(398)] Failed to connect to the bus: Failed to connect to socket /run/firejail/mnt/dbus/system: Permission denied
    signal-desktop[5026]: [15:0805/090744.152979:ERROR:bus.cc(398)] Failed to connect to the bus: Failed to connect to socket /run/firejail/mnt/dbus/system: Permission denied
    signal-desktop[5026]: [15:0805/090744.153009:ERROR:bus.cc(398)] Failed to connect to the bus: Failed to connect to socket /run/firejail/mnt/dbus/system: Permission denied
    signal-desktop[5176]: (node:111) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead

**Comment by remyabel2** [comment]
I encounter this issue as well.

```
Access error: uid 1000, last mount name:/firejail/firejail.ro.dir dir:/run/firejail/firejail.ro.dir type:tmpfs - invalid read-only mount
Access error: uid 1000, last mount name:/ dir:/run/user/1000/doc type:fuse.portal - invalid whitelist mount
Access error: uid 1000, last mount name:/ dir:/run/user/1000/doc type:fuse.portal - invalid noexec mount
```

Except instead of randomly crashes, the application just sometimes refuses to start.

**Comment by ghost** [comment]
It would be helpful if both of you could check if this is similar to any of the other reported issues with [signal-desktop](https://github.com/netblue30/firejail/search?q=signal-desktop&type=issues). If not, create an issue from this discussion and provide as much info as possible (e.g. post full systemd unit etcetera).

**Reply by remyabel2** [reply]
I do not use signal-desktop. It happens with i.e Evolution and KeepassXC when their shortcuts are in `~/.config/xdg/autostart` (which is probably what OP meant by systemd). Before I open an issue though I was planning on testing if it still happens if i use an alternate autostart mechanism and if I can get traces/debug logs.
